legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -655,6 +655,10 @@ class FilingMeta:  # pylint: disable=too-few-public-methods
     @staticmethod
     def display_name(business: Business, filing: FilingStorage) -> Optional[str]:
         """Return the name of the filing to display on outputs."""
+        # if filing is imported from COLIN and has custom disaply name
+        if filing.meta_data and filing.meta_data.get('isLedgerPlaceholder') and\
+                (display_name := filing.meta_data.get('colinDisplayName')):
+            return display_name
```

Comment:
i don't think we can rely on `isLedgerPlaceholder`.  ideally, we eventually remove `isLedgerPlaceholder` when we finish backfilling data for a migrated corp or do it as clean up after all corps are migrated over.  

I think if `colinDisplayName` exists, it should be sufficient condition for returning the colin specific display name.